src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/JudicialDecisionHandler.java

```diff
@@ -418,11 +442,14 @@ private SubmittedCallbackResponse buildConfirmation(CallbackParams callbackParam
                         throw new IllegalArgumentException("Missing data during submission of judicial decision");
                     }
                 } else if (SEND_APP_TO_OTHER_PARTY.equals(requestMoreInfo.getRequestMoreInfoOption())) {
+                    LocalDateTime submissionEndDate = LocalDateTime.parse(caseData
```

Comment:
string parsing to date would have been redundant if it was a date as mentioned in previous comment.